frontend/sw360-portlet/src/main/resources/META-INF/resources/html/utils/includes/importBom.jspf

```diff
@@ -0,0 +1,177 @@
+            $dialog = dialog.open(dialogDivId,
+                {}, // data
+                function(submit, callback) {
+                    // submitCallback
+                },
+                function() {
+                    // beforeShowFn
+
+                    var dialogDiv = $(dialogDivId);
+                    var contentDiv = dialogDiv.find("#spdxBomUploadUpload");
+                    var statusDiv = dialogDiv.find("#spdxBomUploadStatus");
+                    var data = dialogDiv.data();
+                    var portletNamespace = data.portletNamespace;
+
+                    var urls = {};
+                    urls.newAttachment = data.newAttachmentUrl;
+                    urls.uploadAttachmentPart = data.uploadAttachmentPartUrl;
+                    urls.importBom = data.importBomUrl;
+
+                    function getAttachmentIdPromise(file) {
+                        var data = {};
+                        data[portletNamespace + "fileName"] = file.fileName || file.name;
+
+                        return $.ajax({
+                            url: urls.newAttachment,
+                            cache: false,
+                            dataType: 'text',
+                            data: data
+                        });
+                    }
+
+                    function importBomFromAttachment(attachmentContentId) {
+                        var data = {};
+                        data[portletNamespace + "<%=PortalConstants.ATTACHMENT_CONTENT_ID%>"] = attachmentContentId;
+
+                        return $.ajax({
+                            url: urls.importBom,
+                            cache: false,
+                            dataType: "json",
+                            data: data
+                        });
+                    }
+
+                    var r = new Resumable({
+                        target: urls.uploadAttachmentPart,
+                        parameterNamespace: portletNamespace,
+                        simultaneousUploads: 1,
+                        generateUniqueIdentifier: getAttachmentIdPromise,
+                        chunkRetryInterval: 2000,
+                        maxChunkRetries: 3
+                    });
+
+                    r.assignBrowse($('#fileupload-browse')[0]);
+                    r.assignDrop($('#fileupload-drop')[0]);
+
+                    r.on('fileAdded', function (file) {
+                        console.log("fileAdded...");
+                        contentDiv.hide();
+                        statusDiv.show();
+                        r.upload();
+                        statusDiv.html("<h2>Uploading ...</h2>");
+                    });
+                    r.on('fileProgress', function (file) {
+                        console.log("fileProgress...");
+                    });
+                    r.on('fileSuccess', function (file) {
+                        console.log("fileSuccess...");
+                        statusDiv.html("<h2>Importing ...</h2>");
```

Comment:
This "Importing..." and "Import Success" is actully too much left and should be at dame distance form the edge as the text line above ... 

<img width="831" alt="Screenshot 2020-02-19 at 20 12 20" src="https://user-images.githubusercontent.com/3418977/74867029-3d5bb500-5354-11ea-8a74-84ddd8767f35.png">